signac/core/syncedattrdict.py

```diff
@@ -13,14 +13,13 @@
 from collections.abc import MutableMapping
```

Comment:
I noticed that the module-level docstrings for both `synced_list.py` and `synced_dict.py` say that these classes implement `to_base` and `from_base`. However, after our registry-based solution was implemented, only `to_base` needs to be implemented by subclasses because `from_base` uses the `SyncedCollection` implementation that calls the appropriate subclass constructors, so these docstrings need to be updated.